base/process.jl

```diff
@@ -104,14 +104,14 @@ function show(io::IO, cmd::Cmd)
     print_env = cmd.env !== nothing
```

Comment:
Would probably be better to only do this for `show(io::IO, ::MIME"text/plain", cmd::Cmd)`, so as not to affect `repr`.